src/Reader/IterableDataReader.php

```diff
@@ -2,32 +2,26 @@
 
 declare(strict_types=1);
 
-namespace Yiisoft\Data\Reader\Iterable;
+namespace Yiisoft\Data\Reader;
```

Comment:
```suggestion
namespace Yiisoft\Data\Reader\Iterable;
```

Seems, better keep namespace and don't mix common data reader contracts and classes with implementation (iterable reader and specific filter handlers).